README.md

```diff
@@ -1,7 +1,428 @@
-# What is ChatKC
+# The ChatKC Protocol
 
-ChatKC is a specialized WSS (**W**eb**S**ocket **S**ecure)  chat created by the popular youtuber, [MattKC](https://www.youtube.com/mattkc).<br><br>
-ChatKC is different because it uses Google oAuth to authenticate users to the server after sign in.<br><br>
-After the user is authenticated, they may chat on the MattKC site, or on an alternative, unofficial specialized client.<br><br>
-Work is being done to fully reverse-engineer the ChatKC usage, so that others can make a decentralized network of ChatKC-style servers.<br><br>
+ChatKC is a protocol that originated from [MattKC](https://www.youtube.com/@MattKC)'s website [here](https://stream.mattkc.com/) and reverse engineered by [Breadpudding](https://github.com/cbpudding) and [janKonku](https://github.com/zegfarce) on January 16th, 2023. After playing around with the protocol for hours, MattKC found us hanging out and decided to tweak the protocol a bit to make things easier for us to work with. This document describes the protocol that was created as a result.
```

Comment:
good wording!

Reply:
Lol I just renamed a file, it's the original protocol.md file. The negative diff was mine